pkg/cluster/cluster.go

```diff
@@ -390,6 +396,135 @@ func (c *Cluster) createPod(members etcdutil.MemberSet, m *etcdutil.Member, stat
 	return err
 }
 
+// createNetPolicy creates a NetworkPolicy for a Sensu cluster
```

Comment:
Right now the method creates a single NetworkPolicy for all pods with the label `app=sensu`.

I feel like it would be better to maintain a policy per cluster.

Reply:
done